core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1714,7 +1714,7 @@ public SqlOperandCountRange getOperandCountRange() {
       new SqlFunction(
           "SIGN",
           SqlKind.OTHER_FUNCTION,
-          ReturnTypes.ARG0,
+          ReturnTypes.ARG0_OR_EXACT_NO_SCALE,
           null,
```

Comment:
Thanks, can we give a test case, e.g. the SqlvalidatorTest and SqlOperatorBaseTest.

Reply:
+1 for adding test.